docs/reference/filters/pluralize.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Pluralize
+parent: Filters
+grand_parent: Reference
+---
+
+# pluralize
+
+Outputs the singular or plural version of a string based on a given number.
+
+### Example:
+
+{% raw %}
+```liquid
+<h1>{{ items }} {{ items | pluralize: 'item', 'items' }} in your cart</h1>
```

Comment:
Can we use the variable name `item_count` to make this slightly clearer?

```
{{ item_count }} {{ item_count | pluralize: 'item', 'items' }}
```

Reply:
Nice. Will do!